### PR TITLE
Remove docker install instruction.

### LINKED ISSUE #222: [Test] scripts/test.sh may not put the libcls_arrow library to correct location
When following the [contribution guide](https://github.com/ljishen/skyhookdm-arrow/blob/arrow-master/cpp/src/arrow/adapters/arrow-rados-cls/docs/contributing.md), the CentOS environment in `uccross/skyhookdm-arrow:v0.2.0` has the default class lookup dir set to `@CMAKE_INSTALL_LIBDIR@/rados-classes`, which is `/usr/lib64/rados-classes`. This is not the path `/usr/lib/x86_64-linux-gnu/rados-classes/` the `test.sh` script supposes.

To make it portable, we can set the `osd class dir` for [osd] in the `ceph.conf`. This is the same way used by `vstart.sh` in Ceph.

### PR DESCRIPTION
**Pull request by YashJipkate**
Fix #222

### CODE REVIEW

File: cpp/src/arrow/adapters/arrow-rados-cls/docs/contributing.md [hunk 1]
```diff
@@ -32,12 +32,7 @@ cd skyhookdm-arrow/
 docker run -it -v $PWD:/w -w /w --privileged uccross/skyhookdm-arrow:v0.2.0 bash
 ```
 
-3. Install docker.
-```bash
-curl -o- https://get.docker.com | bash
-```
-
-3. Run the build script.
+3. Run the build script. Make sure [docker](https://docs.docker.com/get-docker/) is installed.
```
Comment (ljishen) [inline cpp/src/arrow/adapters/arrow-rados-cls/docs/contributing.md:35]: There is no docker requirement for running the build.sh script. The docker installation should be an implicit requirement on the host if one wants to set up the development environment using docker. I suggest either omitting the "Make sure ....." or putting it at the beginning as a prerequisite for this setup method.

**Review by JayjeetAtGithub — CHANGES_REQUESTED** [review]
@YashJipkate Can you please fix as @ljishen mentioned ? Thanks.

File: cpp/src/arrow/adapters/arrow-rados-cls/docs/contributing.md (hunk 1 above)
Comment (YashJipkate) [reply cpp/src/arrow/adapters/arrow-rados-cls/docs/contributing.md]: Thanks. I have omitted the instruction since I think that step should be obvious.

**Comment by JayjeetAtGithub** [comment]
@YashJipkate Can you please rebase?

File: cpp/src/arrow/adapters/arrow-rados-cls/docs/contributing.md (hunk 1 above)
Comment (JayjeetAtGithub) [reply cpp/src/arrow/adapters/arrow-rados-cls/docs/contributing.md]: Looks good !

**Pull request merged**